docs/sensors/cloudino.md

```diff
@@ -1,9 +1,392 @@
-# Creating your air quality sensor with Cloudino
-
-## What you need to order
-
-## How to install and configure it
-
-## How to add Air Quality sensors
-
-## Register your Cloudino in the IoT Agent
+# Creating your Air Quality Sensor with Cloudino
+
+## Introduction
+
+This guide will help you to create your own air quality sensor unit
+using Cloudino, an IoT (Internet of Things) Agent, to remotely monitor your
+sensors and send your data to the FIWARE platform.
+
+The Cloudino WiFi Connector can be used as a standalone device to directly
+communicate real-life objects to the internet or as an additional microcontroller
+dedicated to the network layer, working in parallel with actual microcontroller
+solutions like Arduino.
+
+## What you need to order
+
+While using with Arduino, the Cloudino WiFi Connector works as another processor
+in parallel dedicated only to the network layer including the IoT protocols,
+leaving the Arduino dedicated to connectivity with sensors and actuators and
+allowing reprogramming Arduino via WiFi or Cloud.
+
+In this manner, we can connect any sensor compatible with Arduino and process our
+data through the Cloudino and FIWARE platforms.
+
+For the purpose of this guide we will use the following components to develop our
+air quality sensor unit:
+
+* Cloudino WiFi Connector (CWC)
+* Arduino UNO
+* PPD42NS. PM10 Sensor
+* DHT11. Temperature / Humidity Sensor
+* MQ-131. Ozone Sensor
+* Grove Multichannel. Gas Sensor (NO2, CO)
+

```

Comment:
can you provide few references where you can acquire the needed components?
especially for cloudino (i suppose it is easy for people to google where to order and arduino, but how do you procure a cloudino?)